src/record/list.js

```diff
@@ -214,7 +226,9 @@ List.prototype._applyUpdate = function( message ) {
 		message.data[ 2 ] = '[]';
```

Comment:
Yup, you can subscribe to lists as records at the moment as well. Once we have proper support for record options we should store a flag in the meta data to differenciate between records and lists